src/py21cmfast/src/HaloBox.c

```diff
@@ -179,93 +351,71 @@ int set_fixed_grids(double redshift, double norm_esc, double alpha_esc, double M
             grids->count[i] = (int)(h_count * prefactor_mass * (1+dens)); //NOTE: truncated
-            grids->halo_sfr_mini[i] = sfr_mini*prefactor_sfr_mini * (1+dens); //zero if !Minihalos
-
-            if(i==0 && user_params_stoc->USE_INTERPOLATION_TABLES){
-                LOG_SUPER_DEBUG("Cell 0 tables: count %.2e mass %.2e nion %.2e sfr %.2e delta %.2f",h_count,mass,nion,sfr,dens);
-                LOG_SUPER_DEBUG("Cell 0 intgrl: count %.2e mass %.2e nion %.2e sfr %.2e",
-                                IntegratedNdM(lnMmin,lnMmax,params,-1,user_params_stoc->INTEGRATION_METHOD_ATOMIC),
-                                IntegratedNdM(lnMmin,lnMmax,params,-2,user_params_stoc->INTEGRATION_METHOD_ATOMIC),
-                                Nion_ConditionalM(growth_z, lnMmin, lnMmax, M_cell, sigma_cell, dens, M_turn_a
-                                            , astro_params_stoc->ALPHA_STAR, alpha_esc, astro_params_stoc->F_STAR10, norm_esc
-                                            , Mlim_Fstar, Mlim_Fesc, user_params_stoc->INTEGRATION_METHOD_ATOMIC),
-                                Nion_ConditionalM(growth_z, lnMmin, lnMmax, M_cell, sigma_cell, dens, M_turn_a
-                                            , alpha_star, 0., norm_star, 1., Mlim_Fstar, 0.
-                                            , user_params_stoc->INTEGRATION_METHOD_ATOMIC));
-                LOG_SUPER_DEBUG("Cell 0 grids: count %d mass %.2e nion %.2e sfr %.2e", grids->count[i],
-                                    grids->halo_mass[i],grids->n_ion[i],grids->halo_sfr[i]);
-                if(flag_options_stoc->USE_MINI_HALOS){
-                    LOG_SUPER_DEBUG("MINI tables: nion %.2e sfr %.2e",nion_mini,sfr_mini);
-                    LOG_SUPER_DEBUG("MINI intgrl: nion %.2e sfr %.2e",
-                                    Nion_ConditionalM_MINI(growth_z, lnMmin, lnMmax, M_cell, sigma_cell,
-                                                            dens, M_turn_m, M_turn_a, alpha_star_mini,
-                                                            alpha_esc, norm_star_mini, norm_esc_mini, Mlim_Fstar_mini,
-                                                            Mlim_Fesc_mini, user_params_stoc->INTEGRATION_METHOD_MINI),
-                                    Nion_ConditionalM_MINI(growth_z, lnMmin, lnMmax, M_cell, sigma_cell,
-                                                dens, M_turn_m, M_turn_a, alpha_star_mini,
-                                                0., norm_star_mini, 1., Mlim_Fstar_mini, 0.,
-                                                user_params_stoc->INTEGRATION_METHOD_MINI));
-                }
-                LOG_SUPER_DEBUG("ACG MTURN %.3e (%.3e) MCG MTURN %.3e (%.3e) REION %.3e",M_turn_a_nofb,M_turn_a,M_turn_m_nofb,M_turn_m,M_turn_r);
-            }
+            grids->halo_mass[i] = mass_intgrl * prefactor_mass * (1+dens);
```

Comment:
not a big thing but `(1+dens)` could be pre-computed and used for all these lines

Reply:
Honestly I think it would hurt readability a little, and it would probably compile to the same thing anyway.

Reply:
OK, fair enough.